advance_windm_edsclrm_module.F90

```diff
@@ -1395,7 +1395,7 @@ subroutine compute_uv_tndcy( solve_type, fcor, perp_wind_m, perp_wind_g, xm_forc
 
       end select
 
-      xm_tndcy(1:gr%nz) = xm_gf(1:gr%nz) + xm_cf(1:gr%nz)  &
+      xm_tndcy(1:gr%nz) = & !xm_gf(1:gr%nz) + xm_cf(1:gr%nz)  &
```

Comment:
would it be less invasive to not change the l_implemented logic an instead just modify the else case:

      xm_tndcy = 0.0_core_rknd

setting it instead to ```xm_tndcy(1:gr%nz) = xm_forcing(1:gr%nz)``` ?